### PR TITLE
Adding the toastmessage and filled up with movietitle and some styling

### PR DESCRIPTION
**Pull request by CarstenJahnke**
Watch Out my new features on my awesome MCU App 🤪

[🚀 get a little insight 🚀](https://mcu-app-git-toastify-carstenjahnke.vercel.app/)

### CODE REVIEW

File: components/Buttons/FavSeenContainer/ButtonSeen/index.js [hunk 1]
```diff
@@ -3,8 +3,51 @@ import { ButtonSeen, ButtonSeenStyle } from "./ButtonsStyle";
...
+  .Toastify__progress-bar--controlled {
+    background-color: gold;
+  }
+
+  .Toastify__emoji {
+    display: inline-block;
+    font-size: 16px;
+    margin-right: 4px;
+  }
+`;
```
Comment (mpagels) [inline components/Buttons/FavSeenContainer/ButtonSeen/index.js:39]: This is code I just saw the second time. Would it be a good idea creating a component out of it and use it here again?

**Review by mpagels — APPROVED** [review]
Hey looks very very good.

I have one small improvement idea. Pls have a look.

File: components/Buttons/FavSeenContainer/ButtonFavorite/index.js [hunk 2]
```diff
@@ -1,10 +1,23 @@
 import { ButtonFav, ButtonFavStyle } from "./ButtonsStyle";
+import { toast } from "react-toastify";
+import { ProgressBar } from "react-toastify";
+import { StyledToastContainer } from "..";
 import Image from "next/image";
 import React, { useState, useEffect } from "react";
 import Star from "./star.svg";
 import StarFilled from "./star-filled.svg";
+import "react-toastify/dist/ReactToastify.css";
```
Comment (nastiche) [inline components/Buttons/FavSeenContainer/ButtonFavorite/index.js:9]: please move your styles somewhere else, we don't use CSS files except from styles.js (global styles)

**Review by LukasWeckner — APPROVED** [review]
That's some serious styles.js file 🙀

Good job ;-)

**Pull request merged**